sopel/config/core_section.py

```diff
@@ -228,17 +203,16 @@ class CoreSection(StaticSection):
 
     """
 
-    ca_certs = FilenameAttribute('ca_certs', default=_find_certs())
-    """The path to the CA certs ``.pem`` file.
+    ca_certs = FilenameAttribute('ca_certs')
+    """The path to the CA certs ``pem`` file.
```

Comment:
Shouldn't this be `PEM`? I get eliminating the `.` because that's not always the file extension (👀 Debian), but the expected _format_ is probably PEM.